nais/prod-gcp.yaml

```diff
@@ -20,6 +20,8 @@ spec:
   ingresses:
     - https://arbeidsgiver.nav.no/altinn-rettigheter-proxy/
     - https://arbeidsgiver.intern.nav.no/altinn-rettigheter-proxy/
+    - https://altinn-rettigheter-proxy.nav.no/altinn-rettigheter-proxy/
```

Comment:
Trenger vi `    - https://altinn-rettigheter-proxy.nav.no/altinn-rettigheter-proxy/`? Den er vel for gcp-trafikk, og der kan vi bruke service discovery i stedet?

Reply:
Den er kanskje grei å ha i dev, da? så slipper vi å legge på access policy hvis noen ønsker å eksperimentere.